tailnet/conn.go

```diff
@@ -49,7 +49,7 @@ import (
 var ErrConnClosed = xerrors.New("connection closed")
 
 const (
-	WorkspaceAgentSSHPort             = 1
+	WorkspaceAgentSSHPort             = 22
```

Comment:
We need to listen on both ports for backward compatibility, else existing CLI versions will be unable to connect!